packages/wonder-blocks-popover/src/components/__tests__/focus-manager.test.tsx

```diff
@@ -109,4 +87,105 @@ describe("FocusManager", () => {
         // Assert
         expect(lastFocusableElementInside).toHaveFocus();
     });
+
+    it("should allow flowing the focus correctly", async () => {
+        // Arrange
+        const externalNodes = (
+            <div>
+                <button>Prev focusable element outside</button>
+                <button>Open popover</button>
+                <button>Next focusable element outside</button>
+            </div>
+        );
+        render(externalNodes);
+
+        // get the anchor reference to be able pass it to the FocusManager
+        const anchorElementNode = screen.getByRole("button", {
+            name: "Open popover",
+        });
+
+        render(
+            <FocusManager anchorElement={anchorElementNode}>
+                <div>
+                    <button>first focusable element inside</button>
+                </div>
+            </FocusManager>,
+        );
```

Comment:
This is cool.  I didn't realize we had tests that used multiple calls to `render()` in this way.